pkg/watch/notify.go

```diff
@@ -17,15 +17,11 @@
 package watch
 
 import (
-	"errors"
 	"expvar"
 	"fmt"
 	"os"
 	"path/filepath"
-	"runtime"
 	"strconv"
-
-	"github.com/tilt-dev/fsnotify"
```

Comment:
I think I started looking at this to see if we still needed this fork; some of the reasons it was needed for were no longer there (upstream merged some fixes), but I don't know where I left off 😂